### PR TITLE
Update READMEs and CONTRIBUTING.md - RSC-1729 RSC-1730

### PR DESCRIPTION
**Pull request by rpokorny**
https://sonatype.atlassian.net/browse/RSC-1729
https://sonatype.atlassian.net/browse/RSC-1730

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,36 +8,85 @@
...
+* If your contribution is a new component, have you added the component to the Server-Side Rendering Tests?
+* Have you added an appropriate version number bump to the lib and gallery `package.json` files?
 
 ## Submitting a PR
 
-* Add the @sonatype/shared-components-group as a reviewer to the PR
-* Try to fix one thing per pull request! Many people work on this code, so the more focused your changes are, the less
-  of a headache other people will have when they merge their work in.
-* Ensure your Pull Request passes tests either locally or via Jenkins (it will run automatically on your PR)
+* Add  as a reviewer to the PR
```
[the remark this answers is not in the document]
Comment (glenhunter) [reply .github/CONTRIBUTING.md:78]: > TODO fill in with steering committee alias

I created an "RSC Steering" team in Github with the three of us as members.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,36 +8,85 @@
...
+contributing teams using Scrum, this is typically the point at which they would add the ticket to a sprint. From here,
+the RSC JIRA flow is similar to that of other projects. After "Ready for Development" are "In Progress", "Waiting for
+Review", "Testing", and "Done". Tickets should be implemented during the "In Progress" phase (see the checklist below)
+and then a pull request submitted for review. PR Review is performed during the "Waiting for Review" phase. See the
+[Submitting a PR](#submitting-a-pr) section below for PR expectations.
 
-### For Non-Sonatypers
+### Merging and Testing
+Once a PR has passed all necessary approvals and checks, it may be merged. On RSC, we do official manual testing _after_
+the PR is merged (though the developer should or course be manually testing their work during development, and reviewers
```
Comment (glenhunter) [inline .github/CONTRIBUTING.md:42]:
```suggestion
the PR is merged (though the developer should of course be manually testing their work during development, and reviewers
```

File: README.md [hunk 3]
```diff
@@ -325,6 +335,11 @@ From gallery/, run `yarn start`.
...
 To view the gallery, point your browser to `http://localhost:4043/`.
 
+## Server-side Rendering Tests
+The `ssr-tests` top-level directory contains a simple next.js project which tests that RSC components can be used
+within next.js without causing server-side errors. Whenever a new component is created, these tests should be updated
+to include that new component. See `ssr-tests/README.md` for details.
+
 ## Continuous Integration and Releases
 This library has CI builds set up for both main and its dev branches. Builds of the main branch automatically get
 released as new versions. As part of this, dev branch builds have enforcement that they must increment the version
```
Comment (glenhunter) [inline README.md:345]: Should we mention that we use semantic versioning? It's implied but not called out directly.

**Review by JasonDSwearingen — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -253,7 +263,7 @@ To update dark mode screenshots, run the following command:
 docker run --rm -it -w /home/jenkins/gallery -v $PWD:/home/jenkins rsc-visualtesting yarn jest-dark -u
 ```
 
-The commands above ensures you update the screenshots within the Docker container. The updates need be run within the
+The commands above ensure you update the screenshots within the Docker container. The updates need be run within the
```
Comment (EmilyBermudez88) [inline README.md:266]:
```suggestion
The commands above ensure you update the screenshots within the Docker container. The updates need to be run within the
```

File: README.md (hunk 3 above)
Comment (rpokorny) [reply README.md:345]: > Should we mention that we use semantic versioning? It's implied but not called out directly.

It says so at the end of this paragraph.  However, also see ab79aff3c

**Review by glenhunter — APPROVED** [review]

**Pull request merged**